src/frontend/js/components/ReferralActivityIndicator/index.tsx

```diff
@@ -159,15 +166,36 @@ export const ReferralActivityIndicator = ({
         />
       );
       break;
+
+    case ReferralActivityVerb.URGENCYLEVEL_CHANGED:
+      message = (
+        <FormattedMessage
+          {...messages[activity.verb]}
+          values={{
+            actorName: getUserFullname(activity.actor),
+          }}
+        />
+      );
+      break;
   }
 
   let messageLine2: React.ReactNode = null;
+  let messageLine3: React.ReactNode = null;
   switch (activity.verb) {
     case ReferralActivityVerb.ANSWERED:
       messageLine2 = (
         <ActivityAnsweredValidations answer={activity.item_content_object} />
       );
       break;
+
+    case ReferralActivityVerb.URGENCYLEVEL_CHANGED:
+      messageLine3 = (
```

Comment:
We could just reuse the `messageLine2` variable as those two cases will never occur simultaneously for the same activity